flask_app/__init__.py

```diff
@@ -6,6 +6,17 @@
 
 dictConfig(LOGGING_CONFIG)
 
+logging.getLogger().setLevel(0)  # Allows all messages to go through logger
```

Comment:
No need to set manually here. Can set in config: 

https://github.com/AI3SW/stargan-v2/blob/b81452e36732eb81b42431c98032038f16a665ec/config.py#L26

Also don't have to set to 0. Just set to debug should work also